mk/checkconf.mk

```diff
@@ -17,7 +17,7 @@ define check-conf-h
 	cnf='$(strip $(foreach var,				\
 		$(call cfg-vars-by-prefix,$1),			\
 		$(call cfg-make-define,$(var))))';		\
-	guard="_`echo $@ | tr -- -/. ___`_";			\
+	guard="_`echo $@ | tr -- -/.+ ___`_";			\
```

Comment:
Minor: one underscore is enough in the second set of characters passed to `tr`. That is:
```
	guard="_`echo $@ | tr -- -/.+ _`_";			\
```